.github/workflows/commentCodeGeneration.ts

```diff
@@ -21,7 +21,14 @@ export async function script(
     issue_number: context.issue.number,
   });
 
-  const body = `Uncommitted changes were detected after runnning <code>generate:*</code> commands.\nPlease run <code>pnpm run preflight</code> to generate/update the related files, and commit them.`;
+  const body = `GitHub Actions has detected that this PR is incomplete.
```

Comment:
**question:** "incomplete" to what?
Maybe we should add a little bit more detail here, like "incomplete with the contribution assumptions" or anything fancier like that

Reply:
I have trouble understanding your question and the suggested alternative.

Incomplete => something is missing.
The next sections explains what the user is supposed to do.

Reply:
And I have trouble understanding the current sentence as it feels incomplete 😅 

How maybe about something like this?

```suggestion
  const body = `GitHub Actions has detected that there are missing steps that needs to be executed before an approval can be given.
```

Reply:
how about "Github Actions found some issues when trying to build the code in this PR."

Reply:
> how about "Github Actions found some issues when trying to build the code in this PR."

I don't like mixing the word issue with such things, because GitHub uses issues already as a term for "ticket"
So often I replace "issue" with "problem", so it can't be confused 
However I find my last proposal still a tiny bit better, as it also provides a clear context/reason/goal of **why** this needs to be tackled.

Reply:
I like the reworded suggestion from @matthewmayer:

> "Github Actions found some problems when trying to build the code in this PR."

@Shinigami92 Your comment suggests that an approval will be given once the CI completes. However, that's not always the case. For instance, if a PR is opened without a connected issue or if we decide against an implementation after multiple review sessions, an approval might not be granted. This is why I lean towards Matthew's suggestion.

Reply:
That's a REALLY good point @xDivisionByZerox 👍

Reply:
So how about this?

```suggestion
  const body = `GitHub Actions raised some problems by trying to process the code in this PR.
```

Reply:
- Issue => problem - ok
- Raise problem - I would rather go for found problems
- To build/processing code - This may be considered more than building/pure code (test snapshots).

From the suggestions so far I like @matthewmayer's modified version the most.

How about
```suggestion
  const body = `GitHub Actions has found problems running the preflight checks?
```
Alternatively,
```suggestion
  const body = `GitHub Actions has found problems processing the PR.
```

Reply:
> How about [...]

Both suggestions are good 👍

Reply:
> How about [...]

I prefer the first suggestion, but without the question mark (`?`) at the end.